src/app/routes/applications/new-role/new-role.component.spec.ts

```diff
@@ -60,22 +60,6 @@ describe('NewRoleComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should be run removePreconditionRole', () => {
-    const index = 1;
-    component.roleForm = fb.group({
-      data: fb.group({
-        enrolmentPreconditions: [
-          [{ type: PreconditionType.Role, conditions: ['a', 'b', 'c', 'd'] }],
-        ],
-      }),
-    });
-    component.removePreconditionRole(index);
-    expect(
-      component.roleForm.get('data').get('enrolmentPreconditions').value[0]
-        .conditions
-    ).toEqual(['a', 'c', 'd']);
-  });
```

Comment:
Angular was complaining on those forms in tests. Also for me, those tests doesn't make sense, because they are not testing component but a specific function. Imo we shouldn't create forms like this in spec files.

Reply:
Ok, sounds good